src/Api/Exceptions/ApiExceptionSubscriber.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Api\Exceptions;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Event\ExceptionEvent;
+use Symfony\Component\HttpKernel\Exception\HttpExceptionInterface;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+class ApiExceptionSubscriber implements EventSubscriberInterface
+{
+  public function onKernelException(ExceptionEvent $event): void
+  {
+    if (0 !== strpos($event->getRequest()->getPathInfo(), '/api')) {
```

Comment:
For readability - str_contains

Reply:
I did this because I wanted to check for /api in the first character of the path. Otherwise a request like /abrahamlincoln/api/ would also land here. 
On the other hand I guess it doesn't matter if such requests land in here.